backend/controllers/users.js

```diff
@@ -181,20 +181,9 @@ const getUser = async (req, res) => {
 };
 
 const getAllUsers = async (req, res) => {
-  console.log(req.params);
   const response = await userModels.findAll();
   if (response) {
-    res.status(200).json({
-      id: response.id,
-      first_name: response.first_name,
-      last_name: response.last_name,
-      email: response.email,
-      postal_code: response.postal_code,
-      city: response.city,
-      country: response.country,
-      phone: response.phone,
-      premium: response.premium
-    });
+    res.json(response);
   } else {
     res.status(404).json({ message: "No users found" });
   }
```

Comment:
That findAll always returns array (may be empty array), so just return that. Also need to catch errors.
Returning an empty array instead of 404 when the table is empty makes handling stuff easier on frontend.
```suggestion
  try {
    const response = await userModels.findAll();
    return res.status(200).json(response);
  } catch (error) {
    return res.status(500).json({ message: "Internal server error" });
  }
```

Reply:
Resolved in commit [1b3ee79](https://github.com/JuhoBjn/online-store/pull/92/commits/1b3ee791f3727433c65c88e63d32209ee2b56d20)